app-libs/exchange-oracle/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
+name = "ita-exchange-oracle"
+version = "0.8.0"
+authors = ["Integritee AG <hello@integritee.network>"]
+edition = "2018"
+
+[features]
+default = ["std"]
+std = [
```

Comment:
Can you move that down? Sorry, I'll buy you ☕ next time  .. but this disturbes my routine comparing if everything has been added as `std`.

Reply:
No worries 😄 done.